.github/ISSUE_TEMPLATE/pip-works--uv-doesn-t.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Pip works, uv doesn't
+about: For cases where `pip install` works but `uv pip install` fails
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Description**
+<!-- If you're using a non-standard python setup, please include some context on your setup. -->
```

Comment:
I feel like people who know that their Python setup is non-standard will probably already put that information in an issue without this prompt, and people who don't know that their Python setup is non-standard probably won't understand what's meant by the term "non-standard" here... Maybe it's better to just ask some concrete questions here? Suggestions:
- How did you install Python? (python.org, pyenv, Windows store, conda, Linux distro, etc.)
- Please give the output of running `python -VV` on the command line

Reply:
Those should be covered by the last two lines, i've removed this and left it as a blanket "description" header

Reply:
makese sense!